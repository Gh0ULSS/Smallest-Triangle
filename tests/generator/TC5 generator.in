-N=16 -mindist=4 -rseed=9