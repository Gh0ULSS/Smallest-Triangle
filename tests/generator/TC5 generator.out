-3.7, -12.67
-36.15, 36.66
-49.36, 0.28
39.83, -41.92
5.43, 11.67
-45.91, -12.1
20.35, -4.8
22.51, -34.28
-26.2, -38.91
0.63, 42.38
9.04, 27.42
-11.63, 24.61
-39.83, -20.88
17.42, 22.57
-7.82, -41.23
-23.33, -29.01
